src/resolvers/group/index.ts

```diff
@@ -0,0 +1,29 @@
+import { UserInputError } from 'apollo-server'
+
+import { QueryResolvers, MutationResolvers } from '../../server-internal-types'
+import { sequelize } from '../../sequelize'
+import Group from '../../models/group'
+
+const groupRepository = sequelize.getRepository(Group)
+
+// Group query resolvers
+const listGroups: QueryResolvers['listGroups'] = async () => {
```

Comment:
question: could this be `QueryResolvers.listGroups` instead or will ts not like that

Reply:
I don't think we can use dot notation unfortunately. Let me look into this more. I don't particularly like it either, but it was the way I could figure out how to get the typings correct. Good news is even with the string access we still get autocomplete and type validation so entering invalid keys will still break the build. But let me research more here...